app/hooks/useLedgerBitcoinWallet.ts

```diff
@@ -97,9 +97,24 @@ export class LedgerBitcoinWallet {
 
   public async sendToAddress(
     address: string,
-    satoshis: string,
-    btcPerKB: number
+    satoshis: string
   ): Promise<string> {
+    const network = await this.getConnectedNetwork();
+
+    // on mainnet, use the wallet's fee estimation
+    // in all other cases, specify a fixed fee
+    const feeOptions =
+      network === 'main'
+        ? {
+            // eslint-disable-next-line @typescript-eslint/camelcase
+            conf_target: 1,
```

Comment:
I propose 6 in https://github.com/comit-network/comit-rs/pull/3256/

1 is going to be unnecessary expensive. It should be align with the expiries.